search/tests/stac/catalog.spec.js

```diff
@@ -50,4 +50,11 @@ describe('createRootCatalog', () => {
     expect(childLink).toBeDefined();
     expect(childLink.href).toBe('/cmr-stac/stac/default');
   });
+
+  it('should be able to add a next rel to pagination links', () => {
+    rootCatalog.addChild('Pagination Link', '/provider');
+    const paginationLink = rootCatalog.links.find((link) => link.rel === 'child');
+    expect(paginationLink).toBeDefined();
+    expect(paginationLink.href).toBe('/cmr-stac/stac/provider');
```

Comment:
Shouldn't the href for the next page end in `/provider/page/:pageNum` based on the example here https://cmr-stac-api.dev.element84.com/stac/catalog/providers/LAADS , or does that get taken care of when using the relative_url in the addNext() function?

Reply:
You were absolutely correct here and I made the correct changes in an update.